recipes-core/glibc/glibc_%.bbappend

```diff
@@ -1,12 +1,2 @@
-# Upstream Glibc does not yet have RISC-V 32-bit support
```

Comment:
perhaps rename the bbappend to glibc_2.32.bbappend so it is disengaged once we upgrade to 2.33 in core, since we will have the support on oe-core then

Reply:
Good idea